protoc-gen-grpc-gateway/httprule/parse_test.go

```diff
@@ -247,7 +253,7 @@ func TestParseSegmentsWithErrors(t *testing.T) {
 	}{
 		{
 			// double slash
-			tokens: []string{"/", eof},
+			tokens: []string{"//", eof},
```

Comment:
Is it worth keeping both of these tests instead of replacing the old one?

Reply:
IIUC the previous test verified that "/" threw an error, and as this PR now allows "/", keeping the previous test would cause it to fail as "/" will no longer throw an error.